imports/ui/components/auth/client/auth.tests.js

```diff
@@ -12,71 +12,62 @@ import {
 } from 'meteor/accounts-base';
 
 import {
+    LoadController
+} from '../../../modules/load/load';
+import {
+    EnsuresLogout,
+    EnsuresUserCreation,
+    EnsuresLogin
+} from '../../../modules/ensure/ensure';
+
+import {
     chai
 } from 'meteor/practicalmeteor:chai';
 
 should();
 
 describe('Auth', function() {
-    let user = {
-        username: 'userCreatedName',
-        password: 'userCreatedPassword',
+    let fakeUser = {
+        username: 'tyrion_lanister',
+        password: 'IDONTCARE',
         create: false,
         _id: ''
     }
 
+    if (!process.env.TESTING)
+        process.env.TESTING = 1;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'process'.](https://www.codacy.com/app/ddspog/MAM-Seed/file/2738956637/issues/source?bid=3340041&fileBranchId=3353847#l38)